Unable to remove cluster

On running `amp cluster remove`, the following error message appears:
```
$ amp cluster remove
[localhost:50101]
ampagent image pull failed, which is expected on a development version
2017/10/16 23:39:28 ampctl (version: 0.16.0-dev, build: 25a17aab)
Nothing found in stack: amp
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x74e505]

goroutine 1 [running]:
github.com/appcelerator/amp/vendor/docker.io/go-docker.(*Client).VolumeList(0x0, 0x10a72a0, 0xc420014050, 0xc4203b2690, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/appcelerator/amp/vendor/docker.io/go-docker/volume_list.go:18 +0x375
github.com/appcelerator/amp/pkg/docker.(*Docker).ListVolumes(0xc42037e810, 0xc4203b2690, 0x9, 0x0, 0x0, 0x109a960, 0xc4201bdc98)
	/go/src/github.com/appcelerator/amp/pkg/docker/volume.go:13 +0x8a
github.com/appcelerator/amp/cluster/ampagent/cmd.removeVolumes(0x10a98c0, 0xc420227200)
	/go/src/github.com/appcelerator/amp/cluster/ampagent/cmd/install.go:350 +0x7a
github.com/appcelerator/amp/cluster/ampagent/cmd.Uninstall(0xc420378b40, 0x11270d8, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/appcelerator/amp/cluster/ampagent/cmd/uninstall.go:36 +0xe6
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).execute(0xc420378b40, 0x11270d8, 0x0, 0x0, 0xc420378b40, 0x11270d8)
	/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:700 +0x47a
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420378480, 0xc420014004, 0xa, 0x1)
	/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:785 +0x30e
github.com/appcelerator/amp/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420378480, 0x3, 0xc420014004)
	/go/src/github.com/appcelerator/amp/vendor/github.com/spf13/cobra/command.go:738 +0x2b
main.main()
	/go/src/github.com/appcelerator/amp/cluster/ampagent/main.go:46 +0x457
2017/10/16 23:39:29 cluster deleted
```

--- a/cluster/ampagent/cmd/uninstall.go
+++ b/cluster/ampagent/cmd/uninstall.go
@@ -19,6 +19,9 @@ func NewUninstallCommand() *cobra.Command {
 func Uninstall(cmd *cobra.Command, args []string) error {
 	stdin, stdout, stderr := term.StdStreams()
 	dockerCli := docker.NewDockerCli(stdin, stdout, stderr)
+	if err := Docker.Connect(); err != nil {
+		return err
+	}
 
 	namespace := "amp"
 	if len(args) > 0 {

--- a/pkg/docker/volume.go
+++ b/pkg/docker/volume.go
@@ -10,6 +10,12 @@ import (
 )
 
 func (d *Docker) ListVolumes(filter opts.FilterOpt) ([]*types.Volume, error) {
+	if d.GetClient() == nil {
+		err := d.Connect()
+		if err != nil {
+			return nil, err
+		}
+	}
 	reply, err := d.client.VolumeList(context.Background(), filter.Value())
 	if err != nil {
 		return nil, err

--- a/pkg/docker/volume.go
+++ b/pkg/docker/volume.go
@@ -11,10 +11,7 @@ import (
 
 func (d *Docker) ListVolumes(filter opts.FilterOpt) ([]*types.Volume, error) {
 	if d.GetClient() == nil {
-		err := d.Connect()
-		if err != nil {
-			return nil, err
-		}
+		return nil, fmt.Errorf("Docker client is not connected")
 	}
 	reply, err := d.client.VolumeList(context.Background(), filter.Value())
 	if err != nil {

